easysid.cpp

```diff
@@ -38,6 +61,40 @@ EXPORT void end_capture()
     }
 }
 
+EXPORT void set_chip(uint32_t chip)
+{
+    if (chip < 0 || chip > 1) {
+        fprintf(stderr, "Invalid SID Chip: %d\n", chip);
+        // TODO: Implement error checking...
+        return;
+    }
+    sid.set_chip_model(static_cast<chip_model>(chip));
+}
+
+EXPORT void set_sampling_parameters(double clock, uint32_t sample_mode, uint32_t sample_freq)
```

Comment:
Where does the `clock` value come from? I see you have a couple of constants, for NTSC and PAL, but they're defined within the `.cpp` file here and are not available to external callers. How is the caller to know what values are appropriate to supply here?